src/ios/AssetBundleDownloader.swift

```diff
@@ -308,7 +308,15 @@ final class AssetBundleDownloader: NSObject, URLSessionDelegate, URLSessionTaskD
       }
 
       do {
-        try fileManager.moveItem(at: location, to: asset.fileURL as URL)
+        // Workaround for https://github.com/meteor/cordova-plugin-meteor-webapp/issues/56
+        // try fileManager.moveItem(at: location, to: asset.fileURL as URL)
+        let linkExists = (try? asset.fileURL.checkResourceIsReachable()) ?? false
+        if !linkExists {
+          try fileManager.moveItem(at: location, to: asset.fileURL as URL)
```

Comment:
I wonder if we might want to replace the existing item if it already exists? In almost all cases the duplicate file will be identical to the existing file (same hash), so the replacement would be redundant… but also safe.